charts/opentelemetry-demo/templates/_helpers.tpl

```diff
@@ -57,3 +57,20 @@ app.kubernetes.io/component: {{ .name}}
 {{- $mergedEnvs = concat $mergedEnvs $envOverrides }}
 {{- tpl (toYaml $mergedEnvs) . }}
 {{- end }}
+
+{{- define "otel-demo.schedulingRules" -}}
```

Comment:
Currently the `.Values.default` values only apply to components and not to jaeger.  My initial gut reaction is to continue to keep them separate (and we can rename the field to `componentDefaults` in a separate PR to denote this).  

The `.Values.observability.jaeger` section could still have `schedulingRules` so that they can be configured and used directly in the jaeger template.

@dmitryax @puckpuck @puckpuck what do you think?

Reply:
I think we should keep components and Jaeger scheduling rules separate.

Reply:
I agree it should be separate. 

But this brings another question: We use sub-charts for the collector, prometheus and grafana. Can we do the same for jaeger? In that case, this PR won't be needed.

Reply:
reverted the code and added jaeger specific logic to generate scheduling rules.

Reply:
@dmitryax i'll resolve this any case you want to keep/discard the PR